lib/veil/credential_collection/chef_secrets_file.rb

```diff
@@ -43,7 +45,7 @@ def initialize(opts = {})
 
         @user    = opts[:user]
         @group   = opts[:group] || @user
-        @version = opts[:version] || 1
+        opts[:version] = CURRENT_VERSION
```

Comment:
The assumption is that any upgrades to the version will happen before this point and at this point we are going to write it out in the CURRENT_VERSION format.